doc/services/storage/zms/zms.rst

```diff
@@ -413,52 +394,49 @@ and this section provides information to help make a choice.
 More generally to make the right choice between NVS and ZMS, all the blockers should be first
 verified to make sure that the application could work with one subsystem or the other, then if
 both solutions could be implemented, the best choice should be based on the calculations of the
-life expectancy of the device described in this section: :ref:`wear-leveling`.
+life expectancy of the device described in this section: `Wear leveling <#wear-leveling>`_.
 
 Recommendations to increase performance
 ***************************************
 
 Sector size and count
 =====================
 
-- The total size of the storage partition should be well dimensioned to achieve the best
-  performance for ZMS.
+- The total size of the storage partition should be set appropriately to achieve the best
+  performance with ZMS.
   All the information regarding the effectively available free space in ZMS can be found
-  in the documentation. See :ref:`free-space`.
-  We recommend choosing a storage partition that can hold double the size of the key-value pairs
+  in the documentation. See `Available space for user data <#Available-space-for-user-data-key-value-pairs>`_.
```

Comment:
This link actually doesn't work (could the documentation build be made to fail in such case @kartben?).
I think it's due to the uppercase. The label doesn't have any.
```suggestion
  in the documentation. See `Available space for user data <available-space-for-user-data-key-value-pairs>`_.
```
Ditto for line 282.